rust/lance-arrow/src/floats.rs

```diff
@@ -69,7 +69,17 @@ pub trait ArrowFloatType {
 }
 
 pub trait FloatToArrayType:
-    Float + Bounded + Sum + AddAssign<Self> + AsPrimitive<f64> + DivAssign + Send + Sync + Copy
+    Float
+    + Bounded
+    + Sum
+    + AddAssign<Self>
+    + AsPrimitive<f64>
+    + DivAssign
+    + Send
+    + Sync
+    + Copy
+    + One
+    + Zero
```

Comment:
I'm confused. `Float` already implies most of these doesn't it?

Reply:
Hmm, yea maybe it actually asks me to include these trait from the caller side? lemme take a look

Reply:
Removed Zero and One